databricks/sdk/dbutils.py

```diff
@@ -163,6 +163,35 @@ def listScopes(self) -> typing.List[SecretScope]:
         return [SecretScope(v.name) for v in self._api.list_scopes()]
 
 
+class _JobsUtil:
+    """Remote equivalent of jobs util"""
+
+    class _TaskValuesUtil:
+        """Remote equivalent of task values util"""
+        values = {}
+
+        def get(self, taskKey: str, key: str, default: any = None, debugValue: any = None) -> None:
+            """
+            Returns the latest task value that belongs to the current job run
+            """
+            return debugValue
```

Comment:
Wow, this API is a bit confusing. What is the difference between `default` and `debugValue`?

Reply:
https://docs.databricks.com/en/dev-tools/databricks-utils.html#get-command-dbutilsjobstaskvaluesget - `default` is returned when we inside job run and task value can't be found. And `debugValue` is what we always return when outside of job: for example, to be able to run notebook outside of job and check if it is just working without errors